pkg/kustomize/kustomize.go

```diff
@@ -21,7 +23,8 @@ const (
 	ManifestsYAML = "fleet-manifests.yaml"
 )
 
-func Process(m *manifest.Manifest, content []byte, dir string) ([]runtime.Object, bool, error) {
+func Process(m *manifest.Manifest, content []byte, options fleet.KustomizeOptions) ([]runtime.Object, bool, error) {
+	dir := options.Dir
 	if dir == "" {
 		dir = "."
 	}
```

Comment:
We may be able to add this directly to the `kustomize` function since we have the build options now.

Reply:
Where do you recommend this `options.Dir` initialization logic be applied?

Reply:
Perhaps on the line below the comment I've added...

```go
func kustomize(fs filesys.FileSystem, buildOptions string) (result []runtime.Object, err error) {
	buildOpts := strings.Split(buildOptions, " ")
	var o build.Options
	cmd := build.NewCmdBuildWithOptions("build", new(bytes.Buffer), &o)
	cmd.Flags().Parse(buildOpts)

        // right here
        dir := options.Dir
	if dir == "" {
		dir = "."
	}
        // above here

	if err := o.Validate([]string{dir}); err != nil {
		return nil, err
	}
	resMap, err := o.RunBuildWithoutEmitter(fs)
	if err != nil {
		return nil, err
	}
	for _, m := range resMap.Resources() {
		result = append(result, &unstructured.Unstructured{
			Object: m.Map(),
		})
	}
	return
}
```

Reply:
@ron1 ugh - I am having one of those weeks it seems. We need it for the multiple function calls. Resolving this comment. Apologies for the misdirection.